rpm/sdl2.spec

```
@@ -42,11 +41,20 @@ to develop SDL applications.
 %autosetup -p1 -n %{name}-%{version}/%{name}
 
 %build
-%configure CFLAGS='-std=c99' --disable-video-x11 --enable-video-wayland --enable-pulseaudio
-%make_build
+mkdir -p build
+pushd build
+%cmake .. \
+  -DLIB_SUFFIX="" \
+  -DPULSEAUDIO=ON \
+  -DVIDEO_WAYLAND=ON \
+  -DVIDEO_X11=OFF \
+  -DWAYLAND_LIBDECOR=OFF
+popd
```

You can build it in current source directory:
```suggestion
%cmake \
  -DLIB_SUFFIX="" \
  -DPULSEAUDIO=ON \
  -DVIDEO_WAYLAND=ON \
  -DVIDEO_X11=OFF \
  -DWAYLAND_LIBDECOR=OFF
```

---

When trying to build in source directory the build fails:
```
CMake Error at CMakeLists.txt:2 (message):
  Prevented in-tree build.  Please create a build directory outside of the
  SDL source code and run "cmake -S some_path -B ."
  from there
```

---

Ok, got it.